### ISSUE TITLE
Mute Button Doesn’t Work

### ISSUE THREAD

**Issue by eTron** [PRProd/HA-Firemote#284]
### Describe the bug

Thank you for your work on this. 🙏 Mute button is not working on Apple TV 4K 3rd gen or whichever is the latest.

### To Reproduce

Yes

### Expected Behavior

No

### Which Type of Device Do You Own?

Apple TV

**Comment by PRProd** [comment]
Greetings @eTron!

This behavior is unfortunate, but expected.  

In the [FAQ](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#faq) section of the [README.md](https://github.com/PRProd/HA-Firemote/blob/main/README.md) file, you'll find a question labeled [Why won't the volume, mute, and/or power buttons work from my Firemote card?](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#why-wont-the-volume-mute-andor-power-buttons-work-from-my-firemote-card)  The third paragraph in that answer states:
<br>
> Due to the way the Apple TV Home Assistant integration is built, an Apple TV style Firemote cannot properly simulate a "Mute" button press.  A [Button Override](https://github.com/PRProd/HA-Firemote#button-overrides) will be required in this case.

<br>

If you visit the Firemote wiki page, you will find some further clarification about why this limitation exists under the [How Does Firemote Send commands to my devices?](https://github.com/PRProd/HA-Firemote/wiki#how-does-firemote-send-commands-to-my-devices) section:
<br>

> ### How Does Firemote Send commands to my devices?
> Firemote isn't an integration, it's just a Home Assistant (lovelace) card that is designed to interact with your media player devices in the most efficient way possible. EVERYTHING that Firemote can do, you could do too through scripts and commands.

During the development cycle of the Firemote Apple TV Support, there were some discussions about the mute button.  If you are interested, you can review the beta tester notes here: #200 

Currently, I'm in a development cycle for [Roku support](https://github.com/PRProd/HA-Firemote/discussions/275) and a few more of these situations are showing up.  To handle that, I'm adding a special pop-up for those unhandled buttons witch will also be applied to the Apple TV's mute button.

![image](https://github.com/PRProd/HA-Firemote/assets/25486599/9ebb0021-e5bf-47bc-8489-a54d8e82be17)

This new popup is expected to be included when the official Roku support is released.

Long story short - Firemote cannot send a mute command to your Apple TV device that is transferred via CEC to your TV or receiver, BUT you can create a [button override](https://github.com/PRProd/HA-Firemote#button-overrides) for the Firemote mute button to send the mute command directly to your audio device.

I hope this helps.
